packages/create-react-native-library/templates/native-common/android/build.gradle

```diff
@@ -183,11 +178,17 @@ def kotlin_version = getExtOrDefault('kotlinVersion')
 <% } -%>
 
 dependencies {
+  if (REACT_NATIVE_MAJOR_VERSION == 0 && REACT_NATIVE_MINOR_VERSION < 71) {
     //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native:+"
+    implementation 'com.facebook.react:react-native:+'
+    // From node_modules
+  } else {
+    implementation 'com.facebook.react:react-android:' + REACT_NATIVE_VERSION
```

Comment:
```suggestion
    implementation 'com.facebook.react:react-android'
```

Reply:
will remove after rc3 is released since currently this is needed to avoid build error.